### PR TITLE
Reorder and condense tabs

### PR DESCRIPTION
**Pull request by jmp3833**

### CODE REVIEW

File: assets/js/analysis/new.controller.js [hunk 1]
```diff
@@ -114,6 +113,9 @@
 
     $scope.onAnalysisClick = function (e) {
       $scope.selectedAnalysis = e.analysis;
+
+      //re-enable the preprocessing tab once an analysis is clicked
+      $scope.preprocessingActive = false;
```
Comment (DanielleGonzalez) [inline assets/js/analysis/new.controller.js]: YOLO

**Comment by kmp3325** [comment]
LGTM!

**Comment by daniellegonzalez** [comment]
**LGTM**

**Comment by vastaj** [comment]
This looks OK

**Pull request merged**